bok_choy/web_app_test.py

```diff
@@ -56,6 +109,11 @@ def setUp(self):
         tags = [self.id()]
         self.browser = browser(tags, self.proxy)
 
+        # Needle uses these attributes for taking the screenshots
+        self.driver = self.get_web_driver()
+        self.driver.set_window_position(0, 0)
```

Comment:
How will this work if the thing being screenshot is far below in the page (i.e. out of view because we've scrolled up)?